README.md

```diff
@@ -82,23 +90,46 @@ python3 manage.py createsuperuser
 
 ### Configure OIDC provider (OIDC auth only)
 
-Basically, you can use an OIDC provider of your choice. However, in this tutorial we will use KeyCloak. Therefore, the
-following is a configuration guide for KeyCloak.
+Basically, you can use an OIDC provider of your choice. However, in this tutorial we will use keycloak (version 23.0.3). Therefore, the
+following is a configuration guide for keycloak.
 
 **Create a new client** with following settings:
 
 | Setting                   | Value                                                       |
 |---------------------------|-------------------------------------------------------------|
 | **Name**                  | entirety (or any name according to your naming conventions) |
-| **Authorization Enabled** | ON                                                          |
+| **Client type** | OpenID Connect  
+| **Authentication flow** | Standard flow                                                          |
 | **valid redirect urls**   | http(s)://your application url/oidc/callback/*              |
 
-**Add client roles** with **composite** to next role in list:
+After creating the client, you should be in the client menu of the just created client. Here, navigate to the **roles tab** and create four roles:
 
 * super_admin
 * server_admin
 * project_admin
 * user
 
-Either add users to roles, or create groups and assign them.
+The names of the roles are not important as such. Yet, it makes sense to give them a meaningful name. Additionally, they need to match with the value of the variables set in the environment file earlier, e. g. "OIDC_PROJECT_ADMIN_ROLE=project_admin".
+We recommend to make the rules **composite**. This means that a user having the role project_admin will also have the user, a user being server_admin will have project_admin and user roles assigned. ![Create roles](/figures/roles-composite.png)
```

Comment:
This means that a user having the role project_admin will also have the user **role**...